packages/utils/src/utils.ts

```diff
@@ -22,8 +23,16 @@ export function deepEqualObjects(a: object, b: object): boolean {
     const aKeys = Object.keys(a);
     if (aKeys.length !== Object.keys(b).length) return false;
     for (const key of aKeys) {
-        if (a[key] !== b[key]) {
-            return false;
+        const aValue = a[key];
+        const bValue = b[key];
+        if (typeof aValue === 'object') {
+            if (!deepEqualObjects(aValue, bValue)) {
```

Comment:
What a "deep equal" function it was before this commit... :)

Reply:
Right? :-D